charts/portworx/templates/portworx-ds.yaml

```diff
@@ -161,6 +169,23 @@ spec:
              ]
            {{- end }}
           env:
+          {{ if .Values.AKSInstall }}
+            - name: AZURE_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: px-azure
```

Comment:
how would a user know to create a secret `px-azure` before trying to do a `helm install`?

Reply:
https://docs.portworx.com/portworx-install-with-kubernetes/cloud/azure/aks/2-deploy-px/#install

Reply:
it's explained here https://docs.portworx.com/portworx-install-with-kubernetes/cloud/azure/aks/2-deploy-px/#create-a-secret-to-give-portworx-access-to-azure-apis

Reply:
Right. My point was, can we also add a note in our helm readme? We cannot expect someone to have landed on our helm docs from docs.portworx.com

Reply:
@trierra Let's add a section in the README.

## Cloud installs

### Installing on AKS

Add details about how to create the secret here.